codecov_cli/services/staticanalysis/__init__.py

```diff
@@ -139,6 +139,31 @@ async def run_analysis_entrypoint(
         )
     else:
         logger.info("All files are already uploaded!")
+    try:
+        external_id = response_json["external_id"]
+        logger.debug(
+            "Sending finish signal to let API know to schedule static analysis task",
+            extra=dict(extra_log_attributes=dict(external_id=external_id)),
+        )
+        upload_url = enterprise_url or CODECOV_API_URL
+        response = requests.post(
+            f"{upload_url}/staticanalysis/analyses/{external_id}/finish",
+            headers={"Authorization": f"Repotoken {token}"},
+        )
+        if response.status_code >= 500:
+            raise click.ClickException("Sorry. Codecov is having problems")
+        if response.status_code >= 400:
+            raise click.ClickException(
+                f"There is some problem with the submitted information.\n{response_json.get('detail')}"
+            )
+    except requests.RequestException:
+        raise click.ClickException(click.style("Unable to reach Codecov", fg="red"))
+    logger.info(
+        "Received response from server",
+        extra=dict(
+            extra_log_attributes=dict(time_taken=response.elapsed.total_seconds())
```

Comment:
I think it makes sense if we logged out the response status code and and the response text, wdyt?

Reply:
makes sense!